packages/core/src/external/carequality/ihe-gateway/outbound-result-pooler-direct.ts

```diff
@@ -8,6 +13,27 @@ import {
 
 const api = axios.create();
 
+export type OutboundPatientDiscoveryRespResults = {
```

Comment:
Thanks for doing this!

nit: response results is a bit convoluted - "isn't response the result"? Could add `Param` or `Payload` as a suffix instead.

Reply:
So im cool with this just want to say i thought about using payload but then when using as a type for processing function that you mentioned originally it didnt seem to make sense in that context

Reply:
Yeah, I see. That's why I suggested `Param` first. 😄